Source/Core/Common/Logging/Log.h

```diff
@@ -58,6 +58,9 @@ enum LOG_TYPE
   WII_IPC,
   WIIMOTE,
 
+  // Adding a Log Category for PrimeHack:
```

Comment:
This is nitpicking but this comment is redundant, the enum is literally named PrimeHack, in the logging types enum. Same goes with line LogManager.cpp:147